app/views/RoomInfoView/components/RoomInfoButtons.tsx

```diff
@@ -93,15 +102,16 @@ export const RoomInfoButtons = ({
 				label={i18n.t(isIgnored ? 'Unignore' : 'Ignore')}
 				iconName='ignore'
 				showIcon={!!renderIgnoreUser}
-				danger
+				danger={!renderReportUser}
 			/>
 			<BaseButton
 				onPress={handleBlockUser}
-				label={i18n.t(`${isBlocked ? 'Unblock' : 'Block'}_user`)}
+				label={i18n.t(`${isBlocked ? 'Unblock' : 'Block'}`)}
 				iconName='ignore'
 				showIcon={!!renderBlockUser}
-				danger
+				danger={!renderReportUser}
```

Comment:
Why these optional danger options?
When a `block` won't be dangerous?

Reply:
> Changing the “block” action to being regular blue, as supposed to red. This is because we consider blocking as something you can revert, as opposed to report. That’s why I wanna reserve the red for the latter.

https://www.figma.com/file/OrB7ukVUqRDpJZ890gzrbH/Report-user?type=design&node-id=58-4009&mode=design&t=wvKTcXEp19aoggT4-4